docs/docs/eslint.md

```diff
@@ -1,33 +1,33 @@
 ---
-title: Using ESLint
+título: Utilización ESLint
 ---
 
-ESLint is an open source JavaScript linting utility. Code linting is a type of static analysis that is frequently used to find problematic patterns. There are code linters for most programming languages, and compilers sometimes incorporate linting into the compilation process.
+ESLint es una utilidad de linting JavaScript de código abierto. El linting de código es un tipo de análisis estático que se utiliza con frecuencia para encontrar patrones problemáticos. Hay código linters para la mayor parte de lenguajes de programación, y los compiladores a veces incorporan linting en el proceso de compilación.
```

Comment:
Hay linters de código para...

Reply:
Hi, I just finished resolving this comment.